layout/_partial/header.ejs

```diff
@@ -19,6 +19,14 @@
         <% } %>
       </nav>
       <nav id="sub-nav">
+        <% if (theme.social){ %>
+		<% for (var i in theme.social) {%>
```

Comment:
we are using space instead of tabs, please apply to the rest of lines (that you committed).